src/main/java/core/basesyntax/RandomColor.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface RandomColor {
```

Comment:
Don't use verbs for class/interface names. In this case, the interface name should not imply an action. Consider renaming it to something like `ColorGenerator` to better represent its purpose as per the checklist item: Don't use verbs for class/interface names.